### DISCUSSION TITLE
Corking dilemma

### DISCUSSION THREAD

**Discussion by sailingwithsandeep** [uNetworking/uWebSockets.js#1034] [Q&A] [answered]
Hey i am building https server with routes and everything. (Later i will be buidling events for uWs as well.)
I just want to send simple json responses in stringify (help of [fast-json-stringify](https://github.com/fastify/fast-json-stringify)) format. Do i need to explicty corking or not??
I read the manual and this discussions threads.
-https://github.com/uNetworking/uWebSockets.js/discussions/909
-https://github.com/uNetworking/uWebSockets.js/discussions/265

I am confused.
Thanks.

**Comment by uNetworkingAB** [comment, marked as answer]
The library will fail with an error if you do corking wrong. If it doesn't fail, you are doing it correctly

**Reply by sailingwithsandeep** [reply]
Still i am just sending one send response and be done with it. So do i need to do it or not?? (Explictly)

**Reply by matiaslopezd** [reply]
@uNetworkingAB there is a plan to throw an error in the version 21? Because now is a warning. I think is the correct path to ensure the correct behavior, but know is hard too ensure our developers remember to use cork.
